.github/workflows/tests.yaml

```diff
@@ -125,6 +125,9 @@ jobs:
           cache: 'pip'
       - name: Configure environment
         run: |
+          echo "::group::clean disk"
+          sudo rm -rf /usr/local/lib/android/
```

Comment:
This might help you find more big things that are preinstalled that we don't want other than the Android SDK:

https://github.com/actions/runner-images/tree/main/images/linux/scripts/installers

And just a general list that's more readable: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md

Some other things we could probably nuke if we need more space include:

* Java (three different versions of the SDK)
* Azure, AWS, Google Cloud CLIs
* Chrome, Chromium, Edge, Firefox, webdrivers for them, and selenium
* .net
* Database servers
* Cached Docker images
* Mono
* Swift

Without checking for any side-effects, a rough change to do most of that would be:

```suggestion
          # Remove Android, Java
          sudo rm -rf /usr/local/lib/android/
          dpkg-query --show *jdk-amd64|cut -f1|xargs sudo apt-get -y purge
          sudo rm -rf /usr/lib/jvm /usr/share/apache-maven-*
          # Remove Swift
          sudo rm -rf /usr/share/swift
          # Remove cloud CLIs
          sudo apt-get -y purge azure-cli google-cloud-cli
          sudo rm -rf /usr/local/aws-cli
          # Remove browsers
          sudo apt-get -y purge google-chrome-stable microsoft-edge-stable firefox
          sudo rm -rf /usr/local/share/chromium
          # Remove webdriver
          sudo rm -rf /usr/local/share/chromedriver-linux64 /usr/local/share/edge_driver /usr/local/share/gecko_driver
          # Remove databases
          sudo apt-get -y purge libpq-dev postgresql-* mysql-client mysql-server libmysqlclient mssql-tools 
          # Remove mono
          sudo apt-get -y purge mono-runtime-common nuget
          # Clear cached docker images
          docker system prune --all
          # Remove unused apt packages
          sudo apt-get autoremove -y
          sudo apt-get clean
          
```

Reply:
Thanks!  The android dir is 14GB, which should double the available disk space from 14GB to 28GB.  If it doesn't work, I'll try these next.